test/e2e/validation_test.go

```diff
@@ -254,3 +262,41 @@ func filterSSMatrix(mat *types.ComMatrix) (*types.ComMatrix, error) {
 
 	return &types.ComMatrix{Matrix: res}, nil
 }
+
+func extractNotUsedAndMissingEPS(csvContent string) (*types.ComMatrix, *types.ComMatrix, error) {
+	notUsedEPS := extractDiffByPrefix(csvContent, "+")
+	missingEPS := extractDiffByPrefix(csvContent, "-")
+
+	notUsedEPSMat, err := types.ParseToComDetailsList([]byte(notUsedEPS), types.FormatCSV)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	missingEPSMat, err := types.ParseToComDetailsList([]byte(missingEPS), types.FormatCSV)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	return &types.ComMatrix{Matrix: notUsedEPSMat}, &types.ComMatrix{Matrix: missingEPSMat}, nil
+}
+
+func extractDiffByPrefix(csvContent string, prefix string) string {
```

Comment:
please add a description here (with an example). also, wdyt about having an enum instead of string? I feel like this is not really about prefixes, but about "types" (missing / unused / etc.)